### ISSUE TITLE
In  master  JRS perf regression test, there are failures in submitMessageLoadTest

### ISSUE THREAD

**Issue by ljianghedera** [hashgraph/hedera-services#781]
**Summary of the defect**
SubmitMessageLoadTest failed in the tests.

**How to reproduce (if possible)**
Please refer to the slack report link: https://hedera-hashgraph.slack.com/archives/CKWHL8R9A/p1605687644314700 and
follow its link to detailed log files.

The error message summary is also listed below

**Service logs (if applicable)**
```
---- HAPIClientValidator FAILED validation ----
<<INFO>>
ClientNode - Suite                    Result
Node 0 - CryptoCreateForSuiteRunner   PASSED
Node 1 - CryptoCreateForSuiteRunner   PASSED
Node 2 - CryptoCreateForSuiteRunner   PASSED
Node 3 - CryptoCreateForSuiteRunner   PASSED
Node 4 - CryptoCreateForSuiteRunner   PASSED
Node 4 - SubmitMessageLoadTest        PASSED
Node 5 - CryptoCreateForSuiteRunner   PASSED
<<ERROR>>
ClientNode - Suite               Result   Failures
Node 0 - SubmitMessageLoadTest   FAILED   RunSubmitMessages
Node 1 - SubmitMessageLoadTest   FAILED   RunSubmitMessages
Node 2 - SubmitMessageLoadTest   FAILED   RunSubmitMessages
Node 3 - SubmitMessageLoadTest   FAILED   RunSubmitMessages
Node 5 - SubmitMessageLoadTest   FAILED   RunSubmitMessages
...
```
**Environment:**
- OS: [e.g. Ubuntu 18.04]
- Java: [e.g. OpenJDK 11.0.4]
- Hedera Services Version: [e.g. 0.0.5]
- HAPI Version: [e.g. 0.0.5]

**Additional Context**
Add any other context about the problem here. Attach any logs here, if applicable.

**Comment by ljianghedera** [comment]
The error is caused by insufficient balance, should be expected error messages:

 accountAmounts { accountID { accountNum: 66205 } amount: 900000000000 } accountAmounts { accountID { accountNum: 66200 } amount: -900000000000 } } }; sigs=sigPair { pubKeyPrefix: "\303" ed25519: "\333\367\322C\023\226\326\267E9\240\034\212G\005\351\211\251n\306\220<\n\\^\232R\225D\332i\351\321\271\241g3\240\263\v)\320\215\332\020\032Y\340\337\261\f\a\024\226>D\307\232^t\"\210\360\t" } sigPair { pubKeyPrefix: "\310" ed25519: "\262T\347\267\277O4\001\364Uk!t\251\032\325\332]B\320>\270\316\366\206\336\023\030W0)\362k\342\027\006\272\236\350\200\265f\243x\026;k\"WR\270\fC\266\024\341\216\275q\235\254\224t\b" }
2020-11-18 03:52:22.559 WARN   199  HapiSpecOperation - 'RunSubmitMessages' - HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66205 <- +900000000000, 0.0.66200 -> -900000000000], tokenTransfers=} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<INSUFFICIENT_ACCOUNT_BALANCE>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[SuiteRunner.jar:?]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:251) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:209) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:236) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:331) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:266) ~[SuiteRunner.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-18 03:52:22.559 WARN   199  HapiSpecOperation - 'RunSubmitMessages' - HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66205 <- +900000000000, 0.0.66200 -> -900000000000], tokenTransfers=} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<INSUFFICIENT_ACCOUNT_BALANCE>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[SuiteRunner.jar:?]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:251) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:209) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:236) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:331) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:266) ~[SuiteRunner.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-18 03:52:22.559 WARN   199  HapiSpecOperation - 'RunSubmitMessages' - HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66205 <- +900000000000, 0.0.66200 -> -900000000000], tokenTransfers=} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<INSUFFICIENT_ACCOUNT_BALANCE>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[SuiteRunner.jar:?]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:251) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:209) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:236) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:331) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:266) ~[SuiteRunner.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-18 03:52:22.559 WARN   199  HapiSpecOperation - 'RunSubmitMessages' - HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66205 <- +900000000000, 0.0.66200 -> -900000000000], tokenTransfers=} failed!
java.lang.AssertionError: Wrong status! expected:<SUCCESS> but was:<INSUFFICIENT_ACCOUNT_BALANCE>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at org.junit.Assert.failNotEquals(Assert.java:834) ~[SuiteRunner.jar:?]
	at org.junit.Assert.assertEquals(Assert.java:118) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:251) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.submitOp(HapiTxnOp.java:209) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:179) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:38) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:236) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:331) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:266) ~[SuiteRunner.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-18 03:52:22.564 WARN   268  HapiApiSpec - 'RunSubmitMessages' - HapiMessageSubmit{sigs=2, payer=sender, node=0.0.3, topic=topic, message=Optional[<ByteString@492234fd size=256>]} failed!
java.lang.AssertionError: Operation 'HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66205 <- +900000000000, 0.0.66200 -> -900000000000], tokenTransfers=}' :: Wrong status! expected:<SUCCESS> but was:<INSUFFICIENT_ACCOUNT_BALANCE>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:40) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:236) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:331) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:266) ~[SuiteRunner.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-18 03:52:22.564 WARN   268  HapiApiSpec - 'RunSubmitMessages' - HapiMessageSubmit{sigs=2, payer=sender, node=0.0.3, topic=topic, message=Optional[<ByteString@703ef2ad size=256>]} failed!
java.lang.AssertionError: Operation 'HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66205 <- +900000000000, 0.0.66200 -> -900000000000], tokenTransfers=}' :: Wrong status! expected:<SUCCESS> but was:<INSUFFICIENT_ACCOUNT_BALANCE>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:40) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:236) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:331) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:266) ~[SuiteRunner.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-18 03:52:22.564 WARN   268  HapiApiSpec - 'RunSubmitMessages' - HapiMessageSubmit{sigs=2, payer=sender, node=0.0.3, topic=topic, message=Optional[<ByteString@3f0f10cc size=256>]} failed!
java.lang.AssertionError: Operation 'HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66205 <- +900000000000, 0.0.66200 -> -900000000000], tokenTransfers=}' :: Wrong status! expected:<SUCCESS> but was:<INSUFFICIENT_ACCOUNT_BALANCE>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:40) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:236) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:331) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:266) ~[SuiteRunner.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-18 03:52:22.564 WARN   268  HapiApiSpec - 'RunSubmitMessages' - HapiMessageSubmit{sigs=2, payer=sender, node=0.0.3, topic=topic, message=Optional[<ByteString@121ad896 size=256>]} failed!
java.lang.AssertionError: Operation 'HapiCryptoTransfer{sigs=2, node=0.0.3, transfers=[0.0.66205 <- +900000000000, 0.0.66200 -> -900000000000], tokenTransfers=}' :: Wrong status! expected:<SUCCESS> but was:<INSUFFICIENT_ACCOUNT_BALANCE>
	at org.junit.Assert.fail(Assert.java:88) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:40) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:47) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.rechargePayerFor(HapiTxnOp.java:264) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.resolveStatus(HapiTxnOp.java:236) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.transactions.HapiTxnOp.finalizeExecFor(HapiTxnOp.java:331) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.infrastructure.DelegatingOpFinisher.finishFor(DelegatingOpFinisher.java:41) ~[SuiteRunner.jar:?]
	at com.hedera.services.bdd.spec.HapiApiSpec.lambda$startFinalizingOps$3(HapiApiSpec.java:266) ~[SuiteRunner.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807) [?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) [?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) [?:?]
	at java.lang.Thread.run(Thread.java:835) [?:?]
2020-11-18 03:52:26.785 INFO   155  RunLoadTest - Thread thread17 ops 241 current TPS 23.930096
2020-11-18 03:52:26.991 INFO   155  RunLoadTest - Thread thread10 ops 251 current TPS 24.977608
2020-11-18 03:52:27.161 INFO   155  RunLoadTest - Thread thread15 ops 257 current TPS 25.6487

**Comment by ljianghedera** [comment]
Should be closed by PR: https://github.com/swirlds/swirlds-platform-regression/pull/677

**Comment by Neeharika-Sompalli** [comment]
Closed by swirlds/swirlds-platform-regression#677
